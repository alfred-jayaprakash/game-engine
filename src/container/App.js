import React from 'react';
import JoinScreen from './JoinScreen';
import GameScreen from './GameScreen';
import AdminScreen from './AdminScreen';
import {Switch, Route} from 'react-router-dom';

const App = () => (
  <div className="app-routes">
    <Switch>
      <Route path="/" exact component={JoinScreen} />
      <Route path="/game" component={GameScreen} />
      <Route path="/admin" component={AdminScreen} />
    </Switch>
  </div>
);

export default App;
