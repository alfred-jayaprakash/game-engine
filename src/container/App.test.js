import React from 'react';
import {shallow} from 'enzyme';
import App from './App';

describe ('App', () => {
  it ('should render correctly without issues', () => {
    const component = shallow (<App />);

    expect (component).toMatchSnapshot ();
  });
});
