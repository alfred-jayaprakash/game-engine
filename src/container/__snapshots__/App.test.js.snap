// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly without issues 1`] = `
<div
  className="app-routes"
>
  <BrowserRouter>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        path="/game"
      />
      <Route
        component={[Function]}
        path="/admin"
      />
    </Switch>
  </BrowserRouter>
</div>
`;
